app/javascript/components/molecules/FormGrid.js

```diff
@@ -22,19 +22,26 @@ FormGrid.Row.defaultProps = {
 
 // Wrapper for each form field.
 // NOTE: Grid is hardcoded. This should not be generic element
-FormGrid.Item = ({ children, small = false, fullWidth = false, medium = false }) => (
-  <div
-    className={classNames(
-      'form-grid_item column',
-      { 'small-6': !(small || fullWidth || medium) },
-      { 'small-3': small },
-      { 'small-4': medium },
-      { 'small-12': fullWidth }
-    )}
-  >
-    {children}
-  </div>
-)
+FormGrid.Item = ({ children, width }) => {
```

Comment:
can you add proptypes to this component that lists out what the options are? here's an example where we specify which values are supported https://github.com/SFDigitalServices/sf-dahlia-lap/blob/main/app/javascript/components/atoms/Button.js#L20